README.md

```diff
@@ -60,7 +60,7 @@ Usage: very_good create <output directory>
 very_good create my_app --desc "My new Flutter app"
 
 # Create a new Flutter app named my_app with a custom org
-very_good create my_app --desc "My new Flutter app" --org "com.custom.org"
+very_good create my_app --desc "My new Flutter app" --org-name "com.custom.org"
```

Comment:
This is actually correct as is. `—org` is an alias so both `—org` and `—org-name` work 👍